### PR TITLE
test: kapa.ai script + widget

### PR DESCRIPTION
**Pull request by nilubava**
## Describe this PR

Testing out kapa.ai widget

### CODE REVIEW

File: src/html.tsx [hunk 1]
```diff
@@ -6,8 +6,17 @@ const HTML = (props: any) => {
...
       <head>
         <meta charSet="utf-8" />
+        {/* Kapa.ai script */}
+        <script defer
+          src="https://widget.kapa.ai/kapa-widget.bundle.js"
+          data-website-id="1b51bb03-43cc-4ef4-95f1-93288a91b560"
+          data-project-name="Prisma"
+          data-project-color="#0B3349"
+          data-project-logo="https://res.cloudinary.com/practicaldev/image/fetch/s--6LfYwHeK--/c_fill,f_auto,fl_progressive,h_320,q_auto,w_320/https://dev-to-uploads.s3.amazonaws.com/uploads/organization/profile_image/1608/0f93b179-76bf-4ee7-a838-e8222fbef062.png"
+          data-button-text="Help"
```
Comment (petradonka) [inline src/html.tsx:16]: @nilubava, FYI, I added this to change the text. I hope it's ok I dabbled in the code :)

File: src/styles/layout.css [hunk 2]
```diff
@@ -494,3 +494,31 @@ a.question::before {
...
+/* Kapa.ai overrides */
+#kapa-widget-container .mantine-Button-root {
+  z-index: 1000;
+}
+
+.mantine-Image-root {
+  height: 2rem;
+}
+
+.mantine-Text-root {
```
Comment (petradonka) [inline src/styles/layout.css]: @nilubava, I also added these overrides, I thought it'd be easier than asking you to tweak these small things... :)

However, the result is not exactly what I had locally through making these exact changes in the inspector. I attached a screenshot. Could you help with fixing, based on the lines I added and this picture?

I thought the smaller size fits better, and I made the border radius consistent with our other similar elements.

CleanShot 2023-09-01 at 19 59 55

Comment (nilubava) [reply src/styles/layout.css]: @petradonka I have added some `!important` tags to force these styles.

Comment (petradonka) [reply src/styles/layout.css]: Easy solution 😄 Thanks!

**Pull request merged**